test/e2e/runner/cli_provisioner.go

```diff
@@ -113,13 +113,15 @@ func (cli *CLIProvisioner) provision() error {
 	}
 	os.Setenv("NAME", cli.Config.Name)
 
-	outputPath := filepath.Join(cli.Config.CurrentWorkingDir, "_output")
-	if !cli.Config.UseDeployCommand {
-		publicSSHKey, err := createSaveSSH(outputPath, cli.Config.Name+"-ssh")
-		if err != nil {
-			return errors.Wrap(err, "Error while generating ssh keys")
+	if cli.Config.PublicSSHKey == "" {
```

Comment:
I think `createSaveSSH` creates both public and private key.
maybe if statement should check if both are passed?